DSharpPlus/DiscordConfiguration.cs

```diff
@@ -86,6 +78,12 @@ public string Token
     /// <para>Defaults to <see cref="Stream"/>.</para>
     /// </summary>
     public GatewayCompressionLevel GatewayCompressionLevel { internal get; set; } = GatewayCompressionLevel.Stream;
+    
+    /// <summary>
+    /// <para>Specifies the probing interval in ms to use when first making requests to the API. Should be slightly higher than your average ping to the discord rest api.</para>
+    /// <para>Defaults to 200 ms</para>
```

Comment:
instead of using para sections, i recommend using `<br/>` (that works); "This should be slightly...", API should be capitalized and the doc comment should end with a dot